router/middleware/session/repo.go

```diff
@@ -157,11 +157,13 @@ func MustPull(c *gin.Context) {
 
 	// debugging
 	if user != nil {
-		c.AbortWithStatus(http.StatusNotFound)
+		c.String(http.StatusNotFound, http.StatusText(http.StatusNotFound))
```

Comment:
@bradrydzewski you got any input on this? Keep current 404 Not found scheme or use 403 with something along the lines of "you do not have permission to read/write to _repo_"?